packages/app/src/request.ts

```diff
@@ -29,9 +31,10 @@ export const getRouteFromApp = ({ middleware }: App, h: Handler<Request, Respons
 export const getProtocol = (req: Request): Protocol => {
   const proto = `http${req.secure ? 's' : ''}`
 
+  /* c8 ignore next */
   if (!trustRemoteAddress(req)) return proto
 
-  const header = (req.headers['X-Forwarded-Proto'] as string) || proto
+  const header = (req.get('X-Forwarded-Proto') as string) || proto
```

Comment:
let's keep `req.headers`

Reply:
`req.headers` was null when I supplied it in this test
https://github.com/aarontravass/tinyhttp/blob/670985580bf4fd0db48d4bfa3c9801813e5be23c/tests/core/request.test.ts#L150

Reply:
hm this is weird, `req.get` does the same thing as `req.headers[...]`

Reply:
![image](https://github.com/tinyhttp/tinyhttp/assets/43901677/d723c0a4-9513-43fb-9f4f-6d0e1051f433)
yeah. dont know why. here's the screenshot